book/graphics.md

```diff
@@ -682,6 +668,18 @@ scrollbar if the whole document fits onscreen.
     screen or the possibility of objects purposefully drawn offscreen.
     In [Chapter 5](layout.md), we'll implement this correctly.
 
+*Emoji*: Add support for emoji to your browser 😀. Emoji are
```

Comment:
Does this render correctly in the print form?

Reply:
Ok, I got this to work but I had to modify the `template-book.tex`